addons/metallb/10_metallb-memberlist-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
I took a look at the original upstream manifest https://raw.githubusercontent.com/metallb/metallb/v0.13.3/config/manifests/metallb-native.yaml and have not found this secret in it at all. Why are these manifest different to that one - and cannot we just use the version from upstream as it is (I would say even without splitting into multiple files, to make maintenance easier)?

Reply:
I also thought about the same thing, I didn't find any secret, but the one upstream has a secret, but no data value.
I would check that out now if it works, merge it all to one file and test it out.